<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_join">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_id">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>아이디를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_pwd">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_id">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_id">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>아이디를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;회원가입&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_pwd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_submit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pwd_chk">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호확인</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>360</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_pwd">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_pwd_chk">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="join_tel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>250</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>'-'를 제외한 숫자로만 입력해 주세요</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>연락처</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그인 성공</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Downloads/pngwing.com(1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
